### PR TITLE
Add new strings for new discussion thread extras

### PR DESCRIPTION
**Pull request by hhovhannes**
Original PRs 
https://github.com/vanilla/vanilla-cloud/pull/8411
https://github.com/vanilla/vanilla-cloud/pull/8468
https://github.com/vanilla/vanilla-cloud/pull/8458
https://github.com/vanilla/vanilla-cloud/pull/8493
https://github.com/vanilla/vanilla-cloud/pull/8531

### CODE REVIEW

**Review by vanilla-dbarbier — APPROVED** [review]
Yes!

File: tx-source/site_core.php [hunk 1]
```diff
@@ -2472,3 +2482,31 @@
...
+$Definition["Tombstone"] = "Tombstone";
+$Definition['Hide OP user info and replace comment with "This content has been removed."'] = 'Hide OP user info and replace comment with "This content has been removed."';
+$Definition["Full"] = "Full";
+$Definition["Completely remove the comment and all associated child comments."] = "Completely remove the comment and all associated child comments.";
+
+
+
+
+
+
```
Comment (s-francis) [inline tx-source/site_core.php:2512]: can we have this extra lines removed

File: tx-source/site_core.php [hunk 2]
```diff
@@ -484,6 +484,7 @@
 $Definition['Connect your profile to social networks.'] = 'Connect your profile to social networks to share activity with your friends and followers.';
 $Definition['Contact Us'] = 'Contact Us';
 $Definition['Content'] = 'Content';
+$Definition['Content from Ignored User.'] = 'Content from Ignored User.';
```
Comment (s-francis) [inline tx-source/site_core.php:487]: why is "from" small cased? should it be "From"?

File: tx-source/site_core.php [hunk 3]
```diff
@@ -1799,6 +1804,7 @@
 $Definition['Subscriber'] = 'Subscriber';
 $Definition['Success'] = 'Success';
 $Definition['Success!'] = 'Success!';
+$Definition['Success! User has been warned.'] = 'Success! User has been warned.';
```
Comment (s-francis) [inline tx-source/site_core.php:1807]: Shouldn't this be split

File: tx-source/site_core.php [hunk 4]
```diff
@@ -2472,3 +2482,31 @@
...
+
+// Discussions and comments bulk actions
+$Definition["You don't have the edit permission on the following discussions:"] = "You don't have the edit permission on the following discussions:";
+$Definition["Selected discussions are already closed."] = "Selected discussions are already closed.";
+$Definition["You don't have the close permission on the following discussions:"] = "You don't have the close permission on the following discussions:";
+$Definition["You don't have the delete permission on the following discussions:"] = "You don't have the delete permission on the following discussions:";
+$Definition["You don't have required permission to warn selected comments."] = "You don't have required permission to warn selected comments.";
+$Definition["Only one user may be warned at a time. Make sure all selected comments are from the same user to proceed with bulk warning."] = "Only one user may be warned at a time. Make sure all selected comments are from the same user to proceed with bulk warning.";
```
Comment (s-francis) [inline tx-source/site_core.php:2492]: I think this should be split as well into two different definitions

File: tx-source/site_core.php [hunk 5]
```diff
@@ -2472,3 +2482,31 @@
...
+$Definition["You have selected <0/> comments."] = "You have selected <0/> comments.";
+$Definition["You don't have required permission to split selected comments."] = "You don't have required permission to split selected comments.";
+$Definition["You don't have required permission to delete selected comments."] = "You don't have required permission to delete selected comments.";
+$Definition["Post has been split."] = "Post has been split.";
+$Definition["Split Comments into New Post"] = "Split Comments into New Post";
+$Definition["View New Post"] = "View New Post";
+$Definition["Posts have been deleted."] = "Posts have been deleted.";
+$Definition["Delete Posts"] = "Delete Posts";
+$Definition["You are about to delete <0/> posts. These posts will remain in the change log. Are you sure you want to continue?"] = "You are about to delete <0/> posts. These posts will remain in the change log. Are you sure you want to continue?";
```
Comment (s-francis) [inline tx-source/site_core.php:2501]: This should split as well

**Review by s-francis — CHANGES_REQUESTED** [review]
Added some suggestions for your review

File: tx-source/site_core.php (hunk 2 above)
Comment (hhovhannes) [reply tx-source/site_core.php:487]: This is correct per Sarah's feedback, see the article https://prowritingaid.com/list-of-words-not-capitalized-in-titles . The rest is addressed.

**Review by s-francis — APPROVED** [review]
LGTM 👍

**Pull request merged**